carsus/io/nuclear/nndc.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+import os
+
+DECAY_DATA_SOURCE_DIR = "/home/syzygy/Desktop/TARDIS/carsus-data-nndc-main/csv"
+
+DECAY_DATA_FINAL_DIR = "/home/syzygy/Downloads/tardis-data/decay-data"
+
+
+class NNDCReader:
+
+    def __init__(self, fname=None):
+        if fname is None:
+            self.fname = DECAY_DATA_SOURCE_DIR
+        else:
+            self.fname = fname
+
+        self._decay_data = None
+
+        # TODO: create an 'nndc_columns' list present in the final dataset.
+
+    @property
+    def decay_data(self):
+        if self._decay_data is None:
+            self._decay_data = self._prepare_nuclear_dataframes()
+        return self._decay_data
+
+    def _get_nuclear_decay_dataframe(self, fpath=None):
```

Comment:
Private methods are typically set with "dunder", a double underscore `__`

Reply:
Same for private properties

Reply:
I understand. I used the single underscore based on the methods in other classes in Carsus which is used to indicate to the developer that a method is supposed to be used internally but no enforcement is carried out from Python's side.

Would you suggest changing them to dunder?

Reply:
If they are single underscore in Carsus then you can maintain that style. Clearly that is something we should discuss as a possible style change to Carsus!